README.md

```diff
@@ -176,6 +176,22 @@ If CORS is activated, these default headers will be sent on every response:
     "Access-Control-Allow-Methods" = "'GET,POST,PUT,DELETE,HEAD,PATCH'"
     "Access-Control-Allow-Headers" = "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'"
 
+### Customizing CORS
```

Comment:
Very good documentation 👍 

Can you add a small example? This makes it easier for many people to get started.